src/pydata_sphinx_theme/assets/styles/sections/_announcement.scss

```diff
@@ -33,6 +33,31 @@
   a {
     color: var(--pst-color-inline-code-links);
   }
+
+  // The "Switch to stable version" link (styled like a button)
+  .button-link-to-stable-version {
+    @include box-shadow();
+
+    $background-color: var(--pst-color-danger);
+    $hover-background-color: var(--pst-color-danger-highlight);
+    $color: var(--pst-color-heading-color);
```

Comment:
Is there a reason to use the heading colour vs the `--pst-color-danger-text` [that we calculate to meet WCAG criteria](https://github.com/pydata/pydata-sphinx-theme/blob/99f9b1c29854ee0f88884e032acc1997dfae370e/src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss#L66-L67)?